src/NuGet/Microsoft.Orleans.Development.nuspec

```diff
@@ -1,34 +0,0 @@
-﻿<?xml version="1.0"?>
```

Comment:
Looked at in hindsight, we actually should have bumped the version number on .Core and .CodeGen packages [only] with PR #194 because of the breaking change to binary API, although we currently don't have any way to do that in build for individual packages -- only all packages.